packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -407,6 +409,30 @@ class SemanticsProperties extends DiagnosticableTree {
   /// Doing so instructs screen readers to not read out the [value].
   final bool obscured;
 
+  /// If non-null, whether the node corresponds to a visually distinct subtree.
+  ///
+  /// When the most specific route semantics node changes, the platform
+  /// accessibility framework will make an edge-triggered announcement and clear
+  /// the current accessibility focus. Specificity is defined as the last child
+  /// in inverse hit-test order.
+  ///
+  /// See also:
+  ///
+  ///  * [routeName] for a description of how the announced value is selected.
+  final bool route;
+
+  /// If non-null, whether the node contains the semantic name for a route.
+  ///
+  /// When the most specific route semantics node changes, the platform will
+  /// search for the first child in inverse hit test order with a routeName 
```

Comment:
hit test order is not fully defined, since it's geometric. Do you mean inverse paint order?

Reply:
Since I'm using the child list in the accessibility bridges, I believe the child lists have already been transformed?  I think yegor also has a pending change which updates the child order here.

The correct choice for scopes route is "what ever is on top", and the correct choice for names route is probably first in the expected semantic order.

https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/view/AccessibilityBridge.java#L727